src/runtime/plugin.ts

```diff
@@ -31,7 +33,29 @@ export default defineNuxtPlugin(async (nuxtApp) => {
 
   // Only fetch session if it was not yet initialized server-side
   if (typeof data.value === 'undefined' && !nitroPrerender && !disableServerSideAuth) {
-    await getSession()
+    // Restore the session data from the cookie
+    const config = useTypedBackendConfig(useRuntimeConfig(), 'local')
```

Comment:
authjs tests are failing because of some type mismatches in this line, I'm not much familiar with typescript, so can you take a look? @zoey-kaiser

Reply:
@Vijayabhaskar96 Can you vive the error ? Maybe we can help.

Reply:
Thanks, but I have identified the problem and fixed it. Tests are passing.